package.json

```diff
@@ -28,6 +28,7 @@
   },
   "resolutions": {
     "immutable": "4.2.3",
+    "protobufjs": "7.2.5",
```

Comment:
This is because a bug introduced in 7.2.6 which have been reported as follows:
https://github.com/protobufjs/protobuf.js/issues/1972
https://github.com/protobufjs/protobuf.js/issues/1975